go.mod

```diff
@@ -32,7 +31,8 @@ require (
 	github.com/mattn/go-isatty v0.0.19
 	github.com/mitchellh/colorstring v0.0.0-20190213212951-d06e56a500db
 	github.com/olekukonko/tablewriter v0.0.5
-	github.com/opencontainers/image-spec v1.0.2
+	github.com/opencontainers/image-spec v1.1.0-rc.3
+	github.com/oras-project/oras-credentials-go v0.2.0
```

Comment:
This dependency is going to be merged into oras-go https://github.com/oras-project/oras-credentials-go/discussions/80

Reply:
It should not take too long:

> We need to wait for couple of weeks or a month to ensure that all APIs are indeed stable. The stability is currently being validated by oras CLI and notation CLI